apps/frontend/src/assets/css/fonts.css

```diff
@@ -0,0 +1,4 @@
+@font-face {
```

Comment:
Let's define font-weights separately, we only need regular, medium, and semi-bold variations. This way it will be much easier to add or remove unneeded variations. Like this:
```
@font-face {
	font-family: Inter;
	font-style: normal;
	font-weight: normal;
	src: url("/assets/fonts/inter/Inter-Regular.ttf") format("truetype");
}
```